### PR TITLE
Enable TLS for the Rest APIs

### PR DESCRIPTION
**Pull request by mrshah-at-work**

### CODE REVIEW

**Comment by mrshah-at-work** [comment]
Signed-off-by: Mihir Shah mrshah@us.ibm.com

**Comment by angrbrd** [comment]
Fixes #683

File: pkg/config/policies.go [hunk 1]
```diff
@@ -219,6 +224,23 @@ func (c *ConfigTx) RemoveOrdererOrgPolicy(orgName, policyName string) error {
 	return removePolicy(c.updated.ChannelGroup.Groups[OrdererGroupKey].Groups[orgName], policyName, policies)
 }
 
+// AddChannelPolicy modifies an existing channel policy configuration.
```
Comment (caod123) [inline pkg/config/policies.go]: Probably should reword this to something like `AddChannelPolicy adds a channel level policy.`
